csrc/scheduler/matmul_heuristic.h

```diff
@@ -90,6 +90,10 @@ class MatmulParams : public HeuristicParams {
   //!    C3 C4 D3 D4
   int grid_swizzle_factor = 1;
 
+  //! swizzle MMA results in shared memory
+  //! coalesced write to global memory
```

Comment:
A nitpick for comment formatting, to be aligned with other comments in the structure:
```suggestion
  //! Swizzle MMA results in shared memory
  //!  coalesced write to global memory
```